flink-core/src/main/java/org/apache/flink/configuration/ConfigConstants.java

```diff
@@ -148,6 +148,16 @@
 	public static final String TASK_MANAGER_NETWORK_DEFAULT_IO_MODE = "taskmanager.network.defaultIOMode";
 
 	/**
+	 * Initial backoff (in ms) for partition requests, which return partition not found.
+	 */
+	public static final String TASK_MANAGER_NETWORK_REQUEST_BACKOFF_INITIAL_MS = "taskmanager.network.requestBackoffInitial";
```

Comment:
Are these config parameters that we want to expose? Or would it be safe to make those internal, if there really is no good case for anyone to ever adjust them?